src/casync-tool.c

```diff
@@ -521,7 +524,11 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_exclude_file = r;
                         break;
 
-                case ARG_UNDO_IMMUTABLE:
+                case ARG_EXCLUDE_FROM:
+                        arg_exclude_from = strdup(optarg);
```

Comment:
missing OOM check.

Reply:
Why strdup? This now causes a leak when the program exits. I think it should be fine to use the optarg here.